flutter_local_notifications/README.md

```diff
@@ -103,11 +274,11 @@ Future selectNotification(String payload) async {
 }
 ```
 
-In the real world, this payload could represent the id of the item you want to display the details of. Once the initialisation has been done, then you can manage the displaying of notifications.
+In the real world, this payload could represent the id of the item you want to display the details of. Once the initialisation is complete, then you can manage the display of notifications.
 
 On iOS, initialisation may show a prompt to requires users to give the application permission to display notifications (note: permissions don't need to be requested on Android). Depending on when this happens, this may not be the ideal user experience for your application. If so, please refer to the next section on how to work around this.
 
-*Notes around initialisation*: if the app had been launched by tapping on a notification created by this plugin, calling `initialize` is what will trigger the `onSelectNotification` to trigger to handle the notification that the user tapped on. An alternative to handling the "launch notification" is to call the `getNotificationAppLaunchDetails` method that is available in the plugin. This could be used, for example, to change the home route of the app for deep-linking. Calling `initialize` will still cause the `onSelectNotification` callback to fire for the launch notification. It will be up to developers to ensure that they don't process the same notification twice (e.g. by storing and comparing the notification id).
+⚠ If the app had been launched by tapping on a notification created by this plugin, calling `initialize` is what will trigger the `onSelectNotification` to trigger to handle the notification that the user tapped on. An alternative to handling the "launch notification" is to call the `getNotificationAppLaunchDetails` method that is available in the plugin. This could be used, for example, to change the home route of the app for deep-linking. Calling `initialize` will still cause the `onSelectNotification` callback to fire for the launch notification. It will be up to developers to ensure that they don't process the same notification twice (e.g. by storing and comparing the notification id).
```

Comment:
I have changed `had` to `has`. What's the difference between using `had` and `has`.

Reply:
I can't explain it well in like an English teacher would (e.g. explaining use of pronouns, nouns etc) but `has` is used here as it's present tense. The best I can do is explain by an example. With "had" there are a bunch of other words that would need other changes that aren't grammatically correct (changes/additions are italicised)

"If the app _had_ been launched by tapping on a notification created by this plugin,  calling `initialize` is what _would've triggered_ the `onSelectNotification` to trigger to handle the notification that the user tapped on"

Since this is a note/warning, using `has` would kept it as present tense that explains what will happen as opposed to what's already happened as `had` is past tense